arch/armv7m/tls_internal.c

```diff
@@ -0,0 +1,28 @@
+/*
+ * Phoenix-RTOS
+ *
+ * libphoenix
+ *
+ * tls_internal.c
+ *
+ * Copyright 2022 Phoenix Systems
+ * Author: Lukasz Leczkowski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#include <sys/types.h>
+
+
+volatile uintptr_t arm_tls_ptr __attribute__((section("armtls"), used)) = 0;
```

Comment:
`attribute used` is not needed.
From gcc documentation:
```
used
    This attribute, attached to a variable with static storage, means that the 
    variable must be emitted even if it appears that the variable is not referenced.
```
That is, this attribute is needed if the variable is defined as static.

Reply:
Ok, thanks, removed it.